### DISCUSSION TITLE
Сборка ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#26] [Announcements]
# Сборка ⚡️ Angular с примерами кода

Конфигурация и сборка Angular приложений перед deploy. Примеры с параметрами --base-href и --deploy-url.

https://angdev.ru/doc/angular-deployment/

**Comment by AlexMihaaa** [comment]
Статья требует обновления. Сейчас уже нету флага --prod
